client/db.go

```diff
@@ -90,7 +89,14 @@ func (kv *KeyValue) ValueBytes() []byte {
 // ValueInt returns the value as an int64. This method will panic if the
```

Comment:
It can still panic if the byte slice is less than 8 bytes long. See https://github.com/cockroachdb/cockroach/blob/master/util/encoding/encoding.go#L193.